sources/packages/backend/libs/sims-db/src/entities/notification.model.ts

```diff
@@ -136,4 +136,8 @@ export enum NotificationMessageType {
    * MSFAA record gets cancelled.
    */
   MSFAACancellation = 12,
+  /**
+   *  OfferingChangeRequest is in progress with student.
```

Comment:
The feature is called "Application offering change request" in an attempt to differentiate from the other "offering request" that can be requested by the institution.
Can we have at least the comment changed to "An Application Offering Change Request is in progress with the student."?